package.json

```diff
@@ -270,7 +270,7 @@
 		"build:analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
 		"build:package-types": "node ./bin/packages/validate-typescript-version.js && ( tsc --build || ( echo 'tsc failed. Try cleaning up first: `npm run clean:package-types`'; exit 1 ) ) && node ./bin/packages/check-build-type-declaration-files.js",
 		"prebuild:packages": "npm run clean:packages && lerna run build",
-		"build:packages": "npm run build:package-types && node ./bin/packages/build.js",
+		"build:packages": "npm run --silent build:package-types && node ./bin/packages/build.js",
```

Comment:
Does it also suppress the error if it happens? Could we run something like `--loglevel error`?

Reply:
> with a standalone script that lives in `bin/` and orchestrates everything.

Sorry I think I missed the convo. Why did we move away from this approach?